.storybook/.babelrc

```diff
@@ -1,9 +1,17 @@
 {
   "presets": ["env", "stage-0", "react"],
   "plugins": [
-    ["transform-runtime", {
-      "polyfill": false,
-      "regenerator": true
-    }]
+    [
+      "react-intl-auto", {
+        "removePrefix": true,
+        "filebase": false,
+        "includeExportName": true,
+        "extractComments": true
+      },
+      "transform-runtime", {
```

Comment:
@mateoclarke can you tell me a bit about what transform runtime does. I want to make sure adding react-intl-auto before it doesn't garble things.

Reply:
I'm not sure. I think I copied the babel config from a storybook docs page but I can't find it. Was expecting to find it here: https://storybook.js.org/configurations/custom-babel-config/

Reply:
👍